extensions/rocksdb-repos/FlowFileRepository.h

```diff
@@ -168,6 +172,8 @@ class FlowFileRepository : public core::Repository, public std::enable_shared_fr
 
  private:
 
+  virtual bool ExecuteWithRetry(std::function<rocksdb::Status()> operation);
```

Comment:
This is just a local helper function, it doesn't need to be `virtual`. Am I missing something?

Reply:
Fair, as it's private, it add no value.